imagemounter/volume.py

```diff
@@ -95,6 +95,10 @@ def fstype(self):
     def mountpoint(self):
         return getattr(self.filesystem, 'mountpoint', None)
 
+    @mountpoint.setter
+    def mountpoint(self, new_mountpoint):
+        self.filesystem.mountpoint = new_mountpoint
+
```

Comment:
Is this really necessary? Can you provide an example for this?

Reply:
This was part of some testing I was doing but is no longer needed. Removed in most recent commit.